packages/server/src/index.ts

```diff
@@ -3,17 +3,12 @@ import {CronCommand, CronJob} from 'cron'
 import {HttpApiServer} from '@blueserver/api/server/http'
 import {cache, updateCacheWithRemote, init as initCache} from '@blueserver/api/cache'
 
-async function loadData() {
-	return updateCacheWithRemote()
```

Comment:
no keep that. when server starts we should always have most recent data, not the data from cache

Reply:
It's wip